src/napari/_vispy/_tests/test_vispy_camera.py

```diff
@@ -17,9 +17,13 @@ def test_camera(make_napari_viewer):
     # updated
     assert viewer.dims.ndisplay == 2
 
-    np.testing.assert_almost_equal(viewer.camera.angles, (0, 0, 90))
+    np.testing.assert_almost_equal(
```

Comment:
```suggestion
    np.testing.assert_almost_equal(viewer.camera.angles, (0,0,0))
    np.testing.assert_almost_equal(
```
Adding this back would help with intent. Especially of the following line